File: c++/Easy/Part002/Print Only Alphabets.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string S;
    getline(cin, S); // Read the entire input string

    string result = ""; // To store the resulting string with only alphabets

    for (char c : S) {
        if (isalpha(c)) { // Check if the character is an alphabet
            result += c; // Append the alphabet character to the result string
        }
    }

    cout << result << endl; // Print the resulting string
    return 0;
}

File: Java/Easy/Part002/Print Only Alphabets.java
import java.util.Scanner;

public class Hello {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read the entire input string
        String S = scanner.nextLine();
        
        // StringBuilder to store the resulting string with only alphabets
        StringBuilder result = new StringBuilder();
        
        // Loop through each character in the string
        for (char c : S.toCharArray()) {
            if (Character.isLetter(c)) { // Check if the character is an alphabet
                result.append(c); // Append the alphabet character to the result StringBuilder
            }
        }
        
        // Print the resulting string
        System.out.println(result.toString());
        
        scanner.close();
    }
}
